.github/workflows/build-test-deploy.yml

```diff
@@ -1055,7 +1055,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v3
         with:
-          tags: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest
+          tags: ${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:latest,${{ env.DOCKER_REGISTRY_NAME }}/subsquid-processor:{{ github.event.pull_request.head.sha }}
           context: subsquid
           push: ${{ github.event_name == 'push' && github.event.pull_request.title_name == 'main' }}
```

Comment:
This pull_request.title is incorrect; you are looking for ref_name (see check-docs job)